x-pack/plugins/ml/public/components/anomalies_table/_anomalies_table.scss

```diff
@@ -75,9 +75,25 @@
     min-width: 150px;
   }
 
+  .mlAnomalyCategoryExamples__link {
+    width: 100%;
+  }
+
   .category-example {
     display: block;
     white-space: nowrap;
+    overflow: hidden;
+    text-overflow: ellipsis;
+  }
+
+  .mlAnomalyCategoryExamples {
+    padding: 2%;
```

Comment:
Does this need to be a percentage, or could you use one of the `$euiSize` variables?

Reply:
Can definitely use the `$euiSize` var 👍